var/spack/repos/builtin/packages/draco/package.py

```diff
@@ -18,6 +18,7 @@ class Draco(CMakePackage):
     maintainers = ['KineticTheory']
 
     version('develop', branch='develop')
+    version('7.13.0',  sha256='7b33e068c3bc59687b92e9e0de917c67c29b5890c7fb3bf9b24ded54610a838a')
```

Comment:
@clevelam You will need to push 7.13.0 to the `github.com/lanl/draco` repository before this will work.

Reply:
Ah.. Will do. Thanks!

Reply:
@clevelam Did you try a spack build with this sha256sum? I think it needs to be `07a443df71d8d3720ced98f86821f714d2bfaa9f17a177c7f0465a59a1e9e719`.  The checksum is for the `.zip` file, not the git hash.

Reply:
sha256sum ~/Draco-draco-7_13_0.zip 
7b33e068c3bc59687b92e9e0de917c67c29b5890c7fb3bf9b24ded54610a838a  /users/clevelam/Draco-draco-7_13_0.zip